deepfence_bootstrapper/main.go

```diff
@@ -67,11 +71,13 @@ func main() {
 	if err != nil {
 		log.Fatal().Msgf("%v", err)
 	}
-	c := make(chan os.Signal, 2)
-	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	os_signals := make(chan os.Signal, 2)
+	signal.Notify(os_signals, os.Interrupt, syscall.SIGTERM)
```

Comment:
you can combine this with context using https://pkg.go.dev/os/signal#NotifyContext